toolchain/check/context.h

```diff
@@ -220,9 +220,16 @@ class Context {
   // Performs a name lookup in a specified scope, returning the referenced
   // instruction. Does not look into extended scopes. Returns an invalid
   // instruction if the name is not found.
+  //
+  // If `for_decl_name` is false, then this is a regular name lookup, and the
+  // name will be poisoned if not found so that later lookups will fail; a
+  // poisoned name will be treated as if it is not declared. Otherwise, this is
+  // a lookup for a name being declared, so the name will not be poisoned, but
```

Comment:
"for_decl_name" making me stumble a little, partly with "is" versus "for". Noting the way you describe it here and how it's called, had you considered something like "is_in_decl" to match "LookupNameInDecl", or "is_declaring", something like that?

Reply:
Changed to `is_being_declared`.